src/setup.py

```diff
@@ -7,9 +7,10 @@
     include_package_data=True,
     install_requires=[
         'pytest==4.3.1',
-        'fire==0.1.3',
+        'pytest-ordering==0.6',
```

Comment:
this stuff, along with mypy, would probably make more sense inside of the CircleCI docker container. let's remove it from here, and also from the `.circleci/config.yml` file?